### PR TITLE
New toggle to disable default VPC creation

### PR DESCRIPTION
**Pull request by cschroer**
New toggles allows us to not create the default VPC.
Useful for projects with specific network configuration like InterConnect attachments.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,4 @@
 .Trashes
 ehthumbs.db
 Thumbs.db
+.terraform.lock.hcl
```
Comment (DatMicha) [inline .gitignore:23]: Looking at https://www.terraform.io/docs/language/dependency-lock.html it seems this file should be commited:
> Terraform has made some changes to the provider dependency selections recorded
in the .terraform.lock.hcl file. Review those changes and commit them to your
version control system if they represent changes you intended to make.

File: variables.tf [hunk 2]
```diff
@@ -50,6 +50,15 @@ variable "enabled_services" {
 /* Network                                                                                        */
 /*                                                                                                */
 /**************************************************************************************************/
+variable "skip_default_vpc_creation" {
+  description = <<-EOD
+    When set to true the module will not create the default VPC or related
+    any NAT or serverless VPC access configuration.
```
Comment (DatMicha) [inline variables.tf:56]: typo? "any related"?

File: .gitignore (hunk 1 above)
Comment (cschroer) [reply .gitignore:23]: Yes for terraform code where us use this module. We are here within the module - we define the dependencies in [version.tf]

**Review by DatMicha — APPROVED** [review]
Looks good to me.

**Pull request merged**